src/sass/downloads/_downloadFilter.scss

```diff
@@ -5,35 +5,29 @@
     margin-top: 1rem;
   }
   .filterCategory {
-    margin-top:1rem;
+    margin-top: 1rem;
   }
   .filterBtn {
-    background-color: white;
+    background-color: #fff;
     border: 1px solid lighten($gray, 10%);
```

Comment:
Need to explicitly set `color` for .filterBtn, the button text currently is white with a white background making it invisible

Reply:
I will address this issues in _SCSS_ for the now.